Makefile

```diff
@@ -19,6 +19,9 @@ proto:
 test: testdeps
 	go test -cpu 1,4 -timeout 7m google.golang.org/grpc/...
 
+testsubmodule: testdeps
+	go test -cpu 1,4 -timeout 7m google.golang.org/grpc/security/advancedtls/...
```

Comment:
I think `go` users go.mod in the current directory, or one in some parent directory (I could be wrong).

Try to cd to the directory and then run go test
```
cd security/advancedtls; go test -cpu 1,4 -timeout 7m google.golang.org/grpc/security/advancedtls/...
```